### ISSUE TITLE
Question: Can training be resumed, and what else is required besides recover_start_step?

### ISSUE THREAD

**Issue by daybreaksly** [inclusionAI/ASearcher#20]
Hi,

I would like to ask whether the training process can be resumed after interruption. If resuming is supported, besides setting recover_start_step, are there any other configurations or considerations I should be aware of (e.g., optimizer state, learning rate schedule, checkpoint settings)?

Thanks in advance!
